_posts/2024-08-05-Debezium-ui-update.adoc

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  Status of Debezium UI
+date:   2024-08-05
+tags: [ debezium, ui, features, kubernetes, operator, debezium-server ]
+author: jcechace
+---
+# Status of Debezium UI
+
+Hello everyone, Jakub here. You may have noticed that there wasn't much happening around Debezium UI lately. 
+This, however, would be only partially true. We own you an explanation in this regard, so please bare with me. 
+
+Let's start with the status of the **current** UI project. It became increasing clear that while UI
+for Debezium is an important part of our vision, developing a UI strictly tied to Kafka Connect is 
+not the right direction. It was thus decied to freeze the development of the **current** web UI 
+project. There were two main reasons for this decision: 
+
+1. The team doesn't have the bandwith (nor the desire) to develop a general purpose Kafka connector UI which would 
+be on par with some of the existing offerings (e.g. Lenses). So our UI would only ever be able to work with Debezium connectors specifically.
+2. Debezium Server is gaining popularity and we would like to offer a user friendly way of managing this flavour of Debezium as well. 
+Unfortunetly, the deployment model of Debezium Server is very different from deploying Debezium as a connector. 
+
+## What's Next?
+As the previous paragraph mentioned the **current** UI, it's probably not a suprise that we did not give up on the idea. Part of the decision
+was to explore a possibility of creating a new UI which while still focused exclusively on Debezium would gradually support multiple deployment
+models. The idea is to start with Debezium Server deployed on Kubernetes, followed by bare metal deployment of Debezium Server.
+At some point in the future we would like to cycle back to Kafka Connect and support also the deployment and operation of Debezium as Kafka
+Connect connectors.
+
+++++
+<div class="imageblock centered-image">
+    <img src="/assets/images/2024-08-05-Debezium-ui-update/architecture.png" class="responsive-image" alt="Architecture">
+</div>
+++++
+
+## Designing the New UI
+One of the goals we defined for ourselves when designing the new platform was to simplify the understanding of what Debezium does.
+For this purpuse we are going to abstract away from the underlying deployment model. Instead of focusing on servers and connectors,
```

Comment:
```suggestion
For this reason, we are going to abstract away from the underlying deployment model. Instead of focusing on servers and connectors,
```